src/html.cr

```diff
@@ -30,7 +30,9 @@ module HTML
   # HTML.escape("Crystal & You") # => "Crystal &amp; You"
   # ```
   def self.escape(string : String) : String
-    string.gsub(SUBSTITUTIONS)
+    io = IO::Memory.new
```

Comment:
It's better (more idiomatic) to use `String.build { |io| escape(string, io) }` here.